README.md

```diff
@@ -1,27 +1,32 @@
-[![Actions Status](https://github.com/nickdesaulniers/github_actions_playground/workflows/Clang%20Linux%20CI%20v2/badge.svg)](https://github.com/nickdesaulniers/github_actions_playground/actions)
+[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/Clang%20Linux%20CI%20v2%20next/badge.svg)](https://github.com/nickdesaulniers/github_actions_playground/actions?query=workflow%3A"Clang+Linux+CI+v2+next")
```

Comment:
I assume the links are still pointing to your repo because we do not have a token for ours?

Reply:
no, copy and paste mistake, good catch, will fix